apps/prairielearn/src/lib/editors.ts

```diff
@@ -973,6 +991,46 @@ export class CourseInstanceCopyEditor extends Editor {
   }
 }
 
+async function updateInfoAssessmentFilesForInstanceCopy(
+  courseInstancePath: string,
+  fromCourseSharingName: string,
+) {
```

Comment:
As written, this function will preserve access rules. I'm not sure if that's desirable, as that could immediately make a bunch of stuff available to students if one isn't careful. cc @mwest1066 for your thoughts.

Reply:
I'm inclined to wipe all access rules on copy. I'm thinking that the usual case will be that someone is copying from a public course instance that has been set up with fully-open access to allow folks to try it out as a student, and they almost certainly won't want this after copying.

Reply:
I agree that makes sense, I just didn't think about it. I'll wipe all access rules.

In the future, we may want to add options for copying over access rules with date adjustments (for people re-using a course semester to semester), but that's incredibly out of scope here.